docsets/Zig/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Zig",
-    "version": "0.8.1",
+    "version": "0.9.0",
     "archive": "zig.tgz",
```

Comment:
These paths are case-sensitive. Please add `zig.tgz` or change the path to `Zig.tgz`.